### ISSUE TITLE
[Change Request]: WW2 Exception for ACE items

### ISSUE THREAD

**Issue by ante185** [official-antistasi-community/A3-Antistasi#3168]
### Is the requested change related to a problem?

In WW2 settings in combination with loading ACE, several items that get added to the rebel's starting arsenal are severely anachronistic, like cellphones and chemlights.
It appears at a glance to be limited to just fn_aceModCompat

### Describe the solution you'd like.

A condition that checks whether to remove the offending item(s) when playing with WW2 factions

**Comment by jaj22** [comment]
This was done at some point. Not sure which version though.

**Comment by jaj22** [comment]
Yeah ok, there's perfectly good exclusion logic there for WW2 stuff but the cellphone and most chemlights aren't in there for some reason.
